src/mame/mame.lst

```diff
@@ -32059,6 +32059,9 @@ nevada                          //
 beezerk                         //
 vlcunk                          //
 
+@source:misc/vocalizer.cpp
+vocalizer
+
```

Comment:
Can you abbreviate the short name to fit it in eight characters to avoid “short name length inflation”?

Reply:
It's fine to keep is as-is, I'm ok with 9 letter shortnames when there's no nice way to abbreviate it. @devinacker your choice.

Reply:
Well, “voclizer” and “vocalizr” both read fine and fit in eight characters.

Reply:
I disagree that those 2 examples are sensible to type when starting mame from cmdline.

Reply:
Can we please relax this "8 letter limit at all costs" suggestion, it's frankly silly for stuff like `megadriv`, I don't want to struggle with MAME CLI suggestions more than enough already.

Reply:
The thing is, you still need to memorise the short names for things you use frequently, and I don’t want to have to type excessively long short names at the command line all the time.  It’s already annoying enough with the length of short names for software lists when you need to type a fully qualified list:item:part identifier.

Reply:
Any word with 1 letter removed is more annoying to type than the full word.
Well, except for "a" or "I" :P

Reply:
But then people will start complaining about things having a letter or two dropped to fit nine characters, and then ten characters, and so on.

You follow this to its conclusion and we end up with short names no longer being short and getting close to the 16 character limit on everything, and then you end up having trouble fitting suffixes for clones again.  Then there’s pressure to extend the limit again.

Short names are supposed to be short identifiers that can be quickly and easily typed, don’t make for excessively long URLs, etc.

Reply:
Is that what you're worried about? It doesn't happen (or if it does, show me where).
When I pick shortnames, my max is 9, except for game&watch with their gnw_ prefix.

Reply:
IMO using a full nine-letter word isn't any less convenient than having to remember to spell it with one fewer vowel or what have you. I'm not really attached to any particular naming, though.